src/heretic/model.py

```diff
@@ -93,6 +98,7 @@ def reload_model(self):
             self.settings.model,
             dtype=dtype,
             device_map=self.settings.device_map,
+            trust_remote_code=self.settings.trust_remote_code,
```

Comment:
This isn't safe in this form because of `evaluate_model` (see my earlier comment).